pr-data.csv

```diff
@@ -1283,17 +1283,23 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalo
 https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.TestPartitionManagement.testPartitionRetention,ID,,,
 https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.utils.TestMetaStoreServerUtils.testGetPartitionspecsGroupedBySDonePartitionCombined,ID,Rejected,https://github.com/apache/hive/pull/2719,
 https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/serde,org.apache.hadoop.hive.serde2.avro.TestAvroDeserializer.canDeserializeTimestamps,OD,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.columnar.TestLazyBinaryColumnarSerDe.testHandlingAlteredSchemas,ID,Rejected,https://github.com/apache/hive/pull/923,
```

Comment:
Do not delete these cases where the status is not empty, e.g., `Rejected` or `Accepted`.

Reply:
Sorry about that, I think I misunderstood the third requirement for contribution